gatsby-cli@next: Error: Cannot find module 'babel-code-frame'

## Description

Using `gatsby-cli@next`, the package `@babel/code-frame` is referred to in the `package.json` using the new npm scoped package notation (with `@`), but it is imported/required in the code using the old `babel-code-frame` notation.

### Steps to reproduce

```
npm install -g gatsby-cli@next

gatsby
```

### Expected result

What should happen?

```
Usage: gatsby <command> [options]
Commands:
  gatsby develop                   Start development server. Watches files,
                                   rebuilds, and hot reloads if something
                                   changes
  gatsby build                     Build a Gatsby project.
  gatsby serve                     Serve previously built Gatsby site.
  gatsby new [rootPath] [starter]  Create new Gatsby project.
Options:
  --verbose      Turn on verbose output               [boolean] [default: false]
  --no-color     Turn off the color in output         [boolean] [default: false]
  -h, --help     Show help                                             [boolean]
  -v, --version  Show version number                                   [boolean]
Pass --help to see all available commands and options.
```

### Actual result

```
$ gatsby
module.js:557
    throw err;
    ^
Error: Cannot find module 'babel-code-frame'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/hsribei/.nvm/versions/node/v9.3.0/lib/node_modules/gatsby-cli/lib/reporter/prepare-stack-trace.js:9:24)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
```

### Environment
gatsby-cli version:
```
  "_resolved": "https://registry.npmjs.org/gatsby-cli/-/gatsby-cli-1.1.41-8.tgz",
  "_shasum": "bfacf30b96edc3129325fdf0c78b2da23e0d489a",
  "_spec": "gatsby-cli@next"
```

Node version: v9.3.0
npm version: 6.1.0
OS: macOS 10.13.4

--- a/packages/gatsby-cli/src/reporter/prepare-stack-trace.js
+++ b/packages/gatsby-cli/src/reporter/prepare-stack-trace.js
@@ -3,7 +3,7 @@
  */
 
 const { readFileSync } = require(`fs`)
-const babelCodeFrame = require(`babel-code-frame`)
+const babelCodeFrame = require(`@babel/code-frame`)
 const stackTrace = require(`stack-trace`)
 const { SourceMapConsumer } = require(`source-map`)
 

--- a/packages/gatsby/src/internal-plugins/query-runner/graphql-errors.js
+++ b/packages/gatsby/src/internal-plugins/query-runner/graphql-errors.js
@@ -1,7 +1,7 @@
 // @flow
 
 import { print, visit, GraphQLError, getLocation } from "graphql"
-import babelCodeFrame from "babel-code-frame"
+import babelCodeFrame from "@babel/code-frame"
 import _ from "lodash"
 import report from "gatsby-cli/lib/reporter"
 

